File: src/main/kotlin/com/huawei/cangjie1/psi/CjContainerNode.java
package com.huawei.cangjie1.psi;

import com.intellij.lang.ASTNode;
import com.intellij.psi.PsiElement;
import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NotNull;


public class CjContainerNode extends CjElementImpl {
    public CjContainerNode(@NotNull ASTNode node) {
        super(node);
    }

    @Override // for visibility
    protected <T> T findChildByClass(Class<T> aClass) {
        return super.findChildByClass(aClass);
    }

    @Override // for visibility
    protected <T extends PsiElement> T findChildByType(IElementType type) {
        return super.findChildByType(type);
    }
}

File: src/main/kotlin/com/huawei/cangjie/psi/CjContainerNode.kt
package com.huawei.cangjie.psi

import com.intellij.lang.ASTNode
import com.intellij.psi.PsiElement
import com.intellij.psi.tree.IElementType

open class CjContainerNode(node: ASTNode) : CjElementImpl(node) {
    public override fun <T> findChildByClass(aClass: Class<T>): T? {
        return super.findChildByClass(aClass)
    }

    public override fun <T : PsiElement > findChildByType(type: IElementType): T? {
        return super.findChildByType(type)
    }
}
